javascript/answers/divyansh-gupta-q7.html

```diff
@@ -0,0 +1,80 @@
+<html>
+    <head>
+
+        <!-- Global site tag (gtag.js) - Google Analytics -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157837769-1"></script>
+        <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            JavaScript | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What are the characteristics of JavaScript ‘Strict Mode’?">
+    </head>
+    <body>
+
+        <div>
+            <img src="../../assets/CN_logo.png" height='72px' alt="Coding Ninjas Logo">
+        </div>
+
+        <div>
+            <p>
+                <small>
+
+                    <a href="../../index.html">Home</a>
+                    &gt; <a href="../index.html">JavaScript Questions</a>
+                    &gt; What are the characteristics of JavaScript ‘Strict Mode’?
+
+                </small>
+               
+            </p>
+        </div>
+
+
+        <div>
+            <div>
+                <h3>
+                    Q: What are the characteristics of JavaScript ‘Strict Mode’?
+                </h3>
+                
+            </div>
+            <div>
+                <p>
+                    ‘Strict mode’ is a restricted variant of JavaScript. <br>
+                    But in ‘Strict mode’ it will throw all types of errors, even the silent errors. 
+                    Thus, the process of debugging becomes easier. And the chances for making mistake for the developer is reduced.
+                </p>
+                <strong>Answer: Given below are the characteristics of JavaScript ‘Strict Mode’:</strong> 
```

Comment:
Write a js program with and without strict mode to make it understand better

Reply:
Yeah I have changed the definition and also added a simple example for it